src/apps/routes/tools.clj

```diff
@@ -16,11 +17,13 @@
                            user-get-tool]]
         [apps.tools.private :only [add-private-tool delete-private-tool update-private-tool]]
         [apps.user :only [current-user]]
+        [apps.util.coercions :only [coerce!]]
         [apps.util.service]
         [slingshot.slingshot :only [throw+]]
         [ring.util.http-response :only [ok]])
   (:require [apps.routes.schemas.permission :as permission]
             [apps.service.apps :as apps]
+            [apps.service.apps.de.listings :as app-listings]
```

Comment:
I was wondering about that. I thought that since tools were specific to the DE (we don't support tools in Agave endpoints) that we didn't need to route anything tool-specific through that interface.

If we might support tools in future 3rd-party services, then this makes sense.